src/main/webapp/app/entities/core/tag/tag-update.component.html

```diff
@@ -1,18 +1,25 @@
-<div class="row justify-content-center">
-    <div class="col-8">
+<div class="row justify-content-center padding-40-0">
+    <div class="col-8 padding-20 white-bc box-shadow-2">
         <form name="editForm" role="form" novalidate (ngSubmit)="save()" #editForm="ngForm">
```

Comment:
use reactive form, not `ngform` see here: https://angular.io/guide/reactive-forms